lib/internal/errors.js

```diff
@@ -1224,3 +1239,6 @@ E('ERR_WORKER_UNSUPPORTED_EXTENSION',
 E('ERR_WORKER_UNSUPPORTED_OPERATION',
   '%s is not supported in workers', TypeError);
 E('ERR_ZLIB_INITIALIZATION_FAILED', 'Initialization failed', Error);
+E('KEYBOARD_INTERRUPT',
```

Comment:
I see no need using the internal error system if this is not going to be captured anyway?